### PR TITLE
Update Student Apply for Sp21 recruitment cycle

### PR DESCRIPTION
**Pull request by suewee**
## Description
Update Student Apply description and timeline to include Interest Form, Application, and Take-Home descriptions and deadlines.

### CODE REVIEW

File: data/studentApplyData.jsx [hunk 1]
```diff
@@ -226,25 +228,33 @@ const studentProcess = [
...
+      </>
+    ),
+    icon: <Contract />,
+    date: `${applicationDueDate}`
+    // date: `${applicationDueDate} ${applicationDueTime}` // due time hidden
+  },
+  {
+    title: "Interviews", //section for for designer recruitment only
+    detail:
+      "Hack4Impact looks for students who are hungry to learn, excellent teammates, and passionate about creating a social impact. Our interviews will give you an opportunity to answer questions about yourself and role-specific tasks. This assingnment is due at midnight.",
```
Comment (arpanlaha) [inline data/studentApplyData.jsx]: What's the last sentence doing here lol

**Review by arpanlaha — CHANGES_REQUESTED** [review]
Just the one comment!

File: data/studentApplyData.jsx (hunk 1 above)
Comment (suewee) [reply data/studentApplyData.jsx]: Oh no good catch :(

**Review by arpanlaha — APPROVED** [review]
🍪

**Pull request merged**